public/app/features/dashboard-scene/saving/provisioned/hooks.ts

```diff
@@ -21,6 +21,8 @@ function generatePath(timestamp: number, pathFromAnnotation?: string, slug?: str
 }
 
 export function useDefaultValues({ meta, defaultTitle, defaultDescription }: UseDefaultValuesParams) {
+  // ???? why is this getting the repository from the folder, not the file???
+  // Should we only get it from the folder when new?
```

Comment:
Hmm, yeah, this needs to be updated. We used to get folder details for a repository to show them in the folder picker.

Reply:
Ill leave this comment here -- but we need to fix all the folder references.  That will make sense after:
https://github.com/grafana/grafana/pull/99791